### DISCUSSION TITLE
Why Ghostty is not using XDG-based configuration location by default on MacOS?

### DISCUSSION THREAD

**Discussion by keinsell** [ghostty-org/ghostty#3503] [Feature Requests, Ideas] [answered]
Today, I installed Ghostty using Homebrew. When I opened the configuration file with `Command` + `,`, I found that the configuration file is located at `/Library/Application Support/com.mitchellh.ghostty/config`. I would like to request a change to `~/.config/ghostty/config`, which I believe would be more appropriate in my opinion and the XDG specification which most terminals are following.

**Comment by mitchellh** [comment, marked as answer]
We support both: https://ghostty.org/docs/config#file-location so feel free to use that. For reasons why we support both, please find the discussion that led to that.

**Reply by keinsell** [reply]
Oh... It's clearly stated. Failed to RTFM.

![image](https://github.com/user-attachments/assets/4ec7c46d-973c-467e-835f-9e7b1dbed862)

By going such also found discussion which explain why such decision was applied, https://github.com/ghostty-org/ghostty/discussions/2567
